From 1d86cc46cd62e72c9d19c7623d984bcbfd896a7f Mon Sep 17 00:00:00 2001
Message-Id: <1d86cc46cd62e72c9d19c7623d984bcbfd896a7f.1587150132.git.zanussi@kernel.org>
In-Reply-To: <05c07aad7517537dec221029782bf891341897c5.1587150132.git.zanussi@kernel.org>
References: <05c07aad7517537dec221029782bf891341897c5.1587150132.git.zanussi@kernel.org>
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Thu, 2 Aug 2018 17:11:01 +0200
Subject: [PATCH 429/506] Revert "arm64/xen: Make XEN depend on !RT"

[ Upstream commit c0a308b58829bd4066bce841fe49e8277a0cb32b ]

Iain Hunter reported that there are no problems with it so there is no
reason to keep it disabled.

Reported-by: Iain Hunter <drhunter95@gmail.com>
Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Signed-off-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
---
 arch/arm64/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 30b28eb72986..a55792388ded 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -792,7 +792,7 @@ config XEN_DOM0
 
 config XEN
 	bool "Xen guest support on ARM64"
-	depends on ARM64 && OF && !PREEMPT_RT_FULL
+	depends on ARM64 && OF
 	select SWIOTLB_XEN
 	select PARAVIRT
 	help
-- 
2.17.1

